mpeg-encoder.js

```diff
@@ -0,0 +1,48 @@
+module.exports = function () {
+  // eslint-disable-next-line
+  importScripts('https://cdnjs.cloudflare.com/ajax/libs/lamejs/1.2.0/lame.min.js')
+
+  var CHANNELS = 1
+  var KBPS = 128
+  var SAMPLE_RATE = 44100
+
+  // eslint-disable-next-line
+  var encoder = new lamejs.Mp3Encoder(CHANNELS, SAMPLE_RATE, KBPS)
+  var recorded = new Int8Array()
+
+  function concat (a, b) {
+    var c = new Int8Array(a.length + b.length)
+    c.set(a)
+    c.set(b, a.length)
+    return c
+  }
+
+  function encode (buffer) {
+    for (var i = 0; i < buffer.length; i++) {
+      buffer[i] = buffer[i] * 32767.5
+    }
+
+    var buf = encoder.encodeBuffer(buffer)
+    if (buf.length > 0) {
+      recorded = concat(recorded, buf)
```

Comment:
CI falls because the test doesn’t cover this line.

(Yeap, it will be the last thing =^_^=)

Reply:
done :)